modules/tinymce/src/plugins/table/test/ts/browser/ClipboardTest.ts

```diff
@@ -732,4 +733,33 @@ describe('browser.tinymce.plugins.table.ClipboardTest', () => {
       '</tbody></table>'
     );
   });
+
+  it('TINY-6939: Pasting into a table should fire TableModified event', () => {
+    const editor = hook.editor();
+
+    let callbackCalled = false;
+    let event: TableEventData = { structure: false, style: false };
```

Comment:
This doesn't accurately test the event is firing correctly, as it only looks at the latest event. This should do what we do elsewhere, whereby we log all the event names/state and then compare them.

Here is an example of one way to do this: https://github.com/tinymce/tinymce/blob/8ee937c891eabef49cb86cce605680b913915346/modules/tinymce/src/plugins/table/test/ts/module/test/TableSizingModeCommandUtil.ts#L74

- [ ] Improve test to ensure only 1 event is fired on paste